### ISSUE TITLE
[Flux Store] - 

### ISSUE THREAD

**Issue by krai500** [inspireui/support#789]
1. My issue
Issue about Snakbar
- Snakbar not show when I login facebook fail
- Snakbar not show 'Welcome' when I login success
 
log
```
flutter: Scaffold.of() called with a context that does not contain a Scaffold.
No Scaffold ancestor could be found starting from the context that was passed to Scaffold.of(). This usually happens when the context provided is from the same StatefulWidget as that whose build function actually creates the Scaffold widget being sought.
There are several ways to avoid this problem. The simplest is to use a Builder to get a context that is "under" the Scaffold. For an example of this, please see the documentation for Scaffold.of():
  https://api.flutter.dev/flutter/material/Scaffold/of.html
A more efficient solution is to split your build function into several widgets. This introduces a new context from which you can obtain the Scaffold. In this solution, you would have an outer widget that creates the Scaffold populated by instances of your new inner widgets, and then in these inner widgets you would use Scaffold.of().
A less elegant but more expedient solution is assign a GlobalKey to the Scaffold
```

3. My screenshot of the purchase product or my comment:
- You can check my username

**Comment by minhcasi** [comment]
Noted for checking, thank you!

**Comment by minhcasi** [comment]
fixed on the latest version v1.3.5 - thank you!
